client/src/screens/burger/view.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Layout, Navbar } from '../../components';
+import { connect } from '../../utils';
+import { controller } from './controller';
+
+class PureBurger extends React.Component {
```

Comment:
Maaan, burger is a name for icon. Burger buttons opens a sceen `Menu`.
Change this name everywhere in the PR